scripts/cleanup_old_vms.py

```diff
@@ -273,7 +274,8 @@ def cleanup_vms(texts, max_hours=24, providers=None, tags=None, prompt=True):
         filters.append(ProviderFilter(keys=providers))
 
     # Just want keys, use list_providers with no global filters to include disabled.
-    providers_to_scan = [prov.key for prov in list_providers(filters, use_global_filters=False)]
```

Comment:
Is it really necessary ? 
Afaik, list_providers doesn't need Appliance.
https://github.com/ManageIQ/integration_tests/blob/master/scripts/ocp_cleanup.py#L79

Reply:
As far as I know, yes. With 'None' as an appliance, scripts will hit iteration errors, and if its completely empty we'll see SSH unavailable failures as it tries to contact an appliance with no IP.

We discussed on rocketchat this behavior in ocp_cleanup and I've included it in the PR now.

Reply:
discussed offline

Reply:
is there any way to us DataProvider objects/only mgt for this - it seems

Reply:
@RonnyPfannschmidt I'm looking now at either adding a method to `list_providers` but for mgmt objects, or adding ProviderFilter support to `list_provider_keys`

Reply:
DataProvider isn't going to be an appropriate approach because of how coupled it is to pytest+metafunc, and builds objects from supportability, not cfme_data.

One additional option that looks viable is to modify ProviderFilter.__call__ implementation to take an AttrDict from cfme_data for a provider, instead of only taking a provider object.  Some small changes would need to be made to the filter functions, but none of the changes look like blockers.

Reply:
I've got some agreement from @RonnyPfannschmidt offline that the other approaches are much more impactful and this is an acceptable solution while we're still doing appliance magic (which we shouldn't be doing)